src/routes.ts

```diff
@@ -3,10 +3,21 @@ import userControllers = require('./controllers/users');
 
 class routes{
 
-    constructor(app:Restify.Server){
+    constructor(server:Restify.Server){
+
+        server.post("/users/login/", userControllers.login);
+        server.get("/users/logout", userControllers.logout);
+        
+        /*********** Documentation routes ***********/
         
-        app.post("/users/login/", userControllers.login);
-        app.get("/users/logout", userControllers.logout);
+        // /docs does not render the css correctly, so redirect to /docs/
+        server.get('/docs', function(req, res, next){
+            res.header('Location', '/docs/');
+            res.send(302);
+        });
+
+        server.get(/\/docs\/?.*/, Restify.serveStatic({directory: './swagger',default: 'index.html'}));
```

Comment:
I just found out you can redirect like this `res.redirect('/docs/', next)`. Also can you add `./swagger` and `index.html` into  `config.ts`?